workers/unity/Packages/io.improbable.gdk.buildsystem/Util/BuildSupportChecker.cs

```diff
@@ -1,11 +1,36 @@
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 using Improbable.Gdk.BuildSystem.Configuration;
 using UnityEditor;
 using UnityEngine;
 
 public static class BuildSupportChecker
 {
+    private static Dictionary<BuildTarget, bool> supportedBuildTargets;
```

Comment:
This could just be a `HashSet` of `BuildTarget`. If the `BuildTarget` isn't present, then it can't be built?